ballerina/http_response.bal

```diff
@@ -517,6 +517,8 @@ public class Response {
             self.setBodyParts(payload);
         } else if payload is stream<SseEvent, error?> {
             self.setSseEventStream(payload);
+        } else if payload is anydata {
```

Comment:
toJson will always create a new value even if the value is already json. May make sense to add an is json check and handle that separately.

Reply:
Yes, that is handled before this check: https://github.com/ballerina-platform/module-ballerina-http/blob/0cb5cf9e14df0023d45b45636d16c2b4c9864652/ballerina/http_response.bal#L512